### ISSUE TITLE
Search box or filter to easily find a substitution in a long list

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Woundorf/foxreplace#134]
```
Hi, I'd like to ask to add a substitution list organization, like organize by 
word, type, replace word, etc. This way, when someone need to edit a word, it 
would be easily found by it's alphabetical order, or if someone need to find a 
word that is case sensitive, it would only need to click on "differentiate 
uppercase/lowercase" and find it easily, or even adding a search box, that 
would be great for people that have a huge substitution list, like me.  =D

Thanks.
```

Original issue reported on code.google.com by `brunolop...@gmail.com` on 27 Feb 2014 at 4:48

**Comment by sqgl2** [comment]
Did this get closed automatically because it is an old issue or is it a feature-request that cannot be implemented for technical reasons?

**Comment by Woundorf** [comment]
It's not closed. This will be when the options are implemented with WebExtensions, which is the thing I'm working on now. It will be available in FoxReplace 1.1.0, which should be ready by summer or sooner if I can.
